BookingsApi/BookingsApi.Infrastructure.Services/Dtos/HearingDto.cs

```diff
@@ -14,5 +14,6 @@ public class HearingDto
         public string HearingVenueName { get; set;}
         public bool RecordAudio { get; set; }
         public string HearingType { get; set; }
+        public string JudgeDisplayName { get; set;}
```

Comment:
You haven't answered why you have added the judge display name to the hearing dto. Is this something needed for every event or just the new allocation hearing event?

Reply:
It is something that maybe we will need in every event

Reply:
Open to a different solution if suggested